raiden/network/matrixtransport.py

```diff
@@ -183,8 +183,9 @@ def start(
         self._join_discovery_room()
         self._discovery_room.add_listener(self._handle_discovery_membership_event, 'm.room.member')
 
+        gevent.spawn_later(1, self._inventory_rooms)
```

Comment:
wouldn't it be better to do this synchronously at the startup? Arbitrary timeouts have tendency to backfire in unexpected situations.

Reply:
This one is called synchronously too a little bit above. I hate these spawn_later too, but `GMatrixClient` requires some time after `start_listener_thread` to sync some things from the server which aren't immediately available, that's why we spawn it here to just do a second pass. It may not be required, is just a safe net to ensure these rooms are liste[ne]d.

Reply:
ok, I haven't noticed that previous call :)